src/services/item/plugins/importExport/itemExportRequest.repository.ts

```diff
@@ -0,0 +1,28 @@
+import { singleton } from 'tsyringe';
+
+import { DBConnection } from '../../../../drizzle/db';
+import { itemRequestExportsTable } from '../../../../drizzle/schema';
+import { ItemRequestExportRaw } from '../../../../drizzle/types';
+
+@singleton()
+export class ItemRequestExportRepository {
+  /**
+   * Create given request export and return it.
+   * @param requestExport RequestExport to create
+   */
+  async addOne(
```

Comment:
Should this be named `createRequest` or `logRequest`, `registerExport`.

`addOne` feels generic. But maybe this is intentional and we want our repository functions to be standard ?